backend/response_handler.py

```diff
@@ -0,0 +1,36 @@
+from flask import jsonify, Response
+from json import dumps
+
+class ResponseHandler:
+    """Class to handle consistent JSON responses for the backend."""
+
+    @staticmethod
+    def success(message: str, data: dict = None) ->Response:
+        """Return a consistent JSON response for success."""
+        response = {
+            "status": "success",
+            "message": message,
+            "data": data or {}
+        }
+        return jsonify(response), 200
+
+    @staticmethod
+    def error(message: str, status_code: int = 400) ->tuple[Response, int]:
```

Comment:
I believe here the return type should be simply a Response type object. Flask understands it carries an error code, but it's not necessary to hint `int` on the return type.